doc/source/actions.rst

```diff
@@ -361,6 +361,16 @@ The ``merge`` action merges the pull request into its base branch. The
        Note that the ``rebase`` method has many drawbacks, see :ref:`strict
        rebase`.
 
+   * - ``strict_method_rebase_prefered_user_login``
```

Comment:
This is a bit short, no? 🤭

Maybe just something as simple as `bot_account` since we might use it for other things in the action if we ever need too. And `bot` make it explicit that it's not for a user but for a `bot`?